packages/in_app_purchase/lib/src/in_app_purchase_connection/product.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+/// The class represents the product object, it combined the common fields in the [SkuDetailsWrapper] and [SKProductWrapper].
+///
+/// You can use this object for simple IAP operations. For more detailed and platform specific IAP operations,
+/// see [SkuDetailsWrapper] and [SKProductWrapper] instead.
+/// The programmer should not need to instantiate this object; rather use after returned by [InAppPurchaseConnection.getProductList].
+class Product {
+  Product({
+    @required this.productIdentifier,
+    @required this.title,
+    @required this.description,
+    @required this.price,
+  });
+
+  /// The ProductIdentifier in App Store Connect or Sku in Google Play console.
+  final String productIdentifier;
```

Comment:
nit: I think `id` would be just as clear here and it's more concise.

Reply:
SG, old habit not using id as a variable name since `id` is a reserved key word in Objective-C